extern/test_bmi_fortran/src/bmi_test_bmi_fortran.f90

```diff
@@ -686,6 +718,30 @@ function test_var_itemsize(this, name, size) result (bmi_status)
     !TODO think of a better way to do this
     !Use 'sizeof' in gcc & ifort
     select case(name)
+    case("grid_1_shape")
```

Comment:
This needs fixed!!! `sizeof` will give you the number of bytes of a scalar, but when called on an array, it gives the number of bytes of the the entire array...